Allow Transformer accept output result that is not Proxy

842974287 opened the pull request:
Summary: Use `map_aggregate` to create output for new graph so that it won't raise error when we have outputs that is not `Proxy`.

Test Plan: `test_transformer_multi_outputs` in `test_fx.py`

Differential Revision: D26502277

jamesr66a commented on torch/fx/experimental/graph_manipulation.py:
```diff
@@ -255,9 +248,31 @@ def serialize_module(fx_module: GraphModule, weights: Dict, name_prefix="") -> D
+            node_rep["args"] = map_arg(
+                node.args, lambda arg: {"is_node": True, "name": str(arg)}
+            )
+
```
Not sure if these changes should be part of this PR?

842974287 replied:
Those changes should belong to another PR. Sorry about this, I'll re-export and land that one first.

842974287 replied:
Looks like it got fixed. Please take a look when you have time @jamesr66a, thanks!

842974287 committed:
Allow Transformer accept output result that is not Proxy (#52473)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/52473

Use `map_aggregate` to create output for new graph so that it won't raise error when we have outputs that is not `Proxy`.

Test Plan: `test_transformer_multi_outputs` in `test_fx.py`

Differential Revision: D26502277

fbshipit-source-id: 3f16f39afc14eea6febf05d177be1112b50ca310

jamesr66a reviewed (approved):
Great! Thanks so much for this
